lib/ansible/modules/packaging/os/yum.py

```diff
@@ -1058,6 +1084,10 @@ def ensure(module, state, pkgs, conf_file, enablerepo, disablerepo,
 
     dis_repos =[]
     en_repos = []
+
+    if skip_broken:
+        yum_basecmd.extend(['--skip-broken'])
```

Comment:
Is skip-broken allowed for all of the commands that we run?  (I've used it with yum update but never with yum remove or yum install).